### PR TITLE
Copy update RE EHR

### PR DESCRIPTION
**Pull request by reshmakh**

### CODE REVIEW

File: packages/docs/docs/medications/e-prescibe.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # E-Prescribe (eRx)
 
-Medplum's flexible architecture allows for integration with any e-prescribe(eRx) provider through the use of [Bots](/docs/bots), offering customizable e-prescription solutions.
+Medplum allows enabling e-prescribe(eRx) functionality.
 
-To provide an integrated prescribing experience out of the box, Medplum offers DoseSpot as its default, first-party eRx provider . While DoseSpot serves as our default solution, [Medplum Bots](/docs/bots) can be used to integrate other prescribing solutions.
+e-Prescribe functionality is only available as part of [Medplum EHR](/solutions/medplum-ehr) and for provider organizations (not for software partners or resellers).
```
Comment (rahul1) [inline packages/docs/docs/medications/e-prescibe.md:5]: Let's make this stronger and make it clear that there is not such thing as a provider organization that can redistribute. Also, wrap in an admonition to make it stand out

```suggestion
::: warning
Medplum's e-prescribe functionality is exclusively available to providers directly engaged in patient care as part of [Medplum EHR](/solutions/medplum-ehr) . Applications that embed or resell this functionality are strictly prohibited. 
:::
```

File: packages/docs/docs/integration/dosespot.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-Medplum has partnered with [DoseSpot](https://www.dosespot.com/), a leader in e-prescription (eRx) technology, to offer prescription ordering services through the Medplum platform.
+Medplum has partnered with [DoseSpot](https://www.dosespot.com/), a leader in e-prescription (eRx) technology, to offer prescription ordering services for the [Medplum EHR](/solutions/medplum-ehr).
 
 This collaboration enables healthcare professionals to seamlessly place medication orders using DoseSpot's eRx interface, which Medplum integrates through an embedded iFrame. This approach ensures a secure, safe, and efficient, experience for clinicians, and offers functionality such as:
 
 - Clinician identity proofing
 - Drug-drug interactions
 - Patient coverage benefits checks
 
+e-Prescribe functionality is only available for provider organizations and not for software partners or resellers.
```
Comment (rahul1) [inline packages/docs/docs/integration/dosespot.md:11]: Let's make this stronger and make it clear that there is not such thing as a provider organization that can redistribute. Also, wrap in an admonition to make it stand out

```suggestion
::: warning
Medplum's e-prescribe functionality is exclusively available to providers directly engaged in patient care as part of [Medplum EHR](/solutions/medplum-ehr) . Applications that embed or resell this functionality are strictly prohibited. 
:::
```

File: packages/docs/docs/home.md [hunk 3]
```diff
@@ -10,13 +10,13 @@ import HomepageCallout from '@site/src/components/HomepageCallout'
 <section className="homepage-grid">
     <HomepageCallout title="Get Started" body="Write your first medical application in 5 minutes" linkText="Read more" linkRef="./docs/tutorials" />
     <HomepageCallout title="API Docs" body="Reference documentation about Medplum's client API for reading and writing healthcare data" linkText="Read more" linkRef="./docs/api" />
-    <HomepageCallout title="Solutions" body="See how to apply Medplum against your healthcare problem" linkText="Read more" linkRef="./solutions" />
+    <HomepageCallout title="Medplum EHR" body="A strealined EHR that's easy to implement" linkText="Read more" linkRef="./solutions/medplum-ehr" />
```
Comment (codyebberson) [inline packages/docs/docs/home.md:13]: I'm ok with this, but this is not what I had imagined during offline discussions.  My memory was that we would keep the "Solutions" title, but include "Medplum EHR" in the body.

**Review by codyebberson — COMMENTED** [review]
Agree with @rahul1 comments, then lgtm 👍

**Review by rahul1 — APPROVED** [review]

**Pull request merged**